src/c++/library/grpc_client.cc

```diff
@@ -322,6 +324,16 @@ InferResultGrpc::IsFinalResponse(bool* is_final_response) const
   return Error::Success;
 }
 
+Error
+InferResultGrpc::IsNullResponse(bool* is_null_response) const
+{
+  if (is_null_response == nullptr) {
```

Comment:
is this missing an underscore?
is_null_response_ since that is holding the value from triton?

Reply:
If not, im not sure I understand this method.

Reply:
it copies the data member into the output pointer